polymetis/polymetis/src/polymetis_server.cpp

```diff
@@ -163,14 +185,6 @@ PolymetisControllerServerImpl::ControlUpdate(ServerContext *context,
     rs_motor_torques_external_[i] = robot_state->motor_torques_external(i);
   }
 
-  // Select controller
```

Comment:
Why is it necessary to move this above parsing the robot state? Just trying to understand what's happening